composer.json

```diff
@@ -13,8 +13,8 @@
         "php": ">=7.2.5",
         "ext-json": "*",
         "paragonie/sodium_compat": "^1.13",
-        "symfony/validator": "^5.1",
-        "guzzlehttp/guzzle": "^7.2",
+        "symfony/validator": "^4.4 || ^5.1",
```

Comment:
Drupal has an agreement with the Symfony security team that we work with them to provide security coverage for old minor versions of components we use, and we have several Symfony 5.0 components in Drupal 9.0 that we might need to make sure work with this project. Let's see if we can allow Symfony 5.0 for now.

Reply:
Symfony 3.4 is actively supported by Symfony until November of this year, but I am fine with a followup for that.

Reply:
Tests are passing with ^5 as the Symfony constraint, so that is probably a good sign. I'm marking this as resolved for now.